src/main/docs/resources/css/custom.css

```diff
@@ -427,3 +427,56 @@ html .dark-mode, body.dark-mode, .dark-mode #main, .dark-mode .docs-content {
     scrollbar-color: #555 rgba(0, 0, 0, 0.3);
     scrollbar-width: thin;
 }
+
+
+@media only screen and (max-width: 820px) {
```

Comment:
Elsewhere we use 768px as the threshold, ie: here

https://github.com/micronaut-projects/micronaut-docs/blob/47fb4b4e8850735315be84cbccd38fb466b62829/src/main/docs/resources/css/custom.css#L134-L141

I'm not sure which is best

Reply:
768 isn't a good choice here from my point of view, I think it is better to use viewports from the most common devices, as 768px originally was for desktop, it was not for mobile devices. I actually don't remember where I got 820px, I have changed it to this selector in the new PR:

```
@media only screen and (max-width: 428px) and (orientation: portrait), only screen and (max-width: 926px) and (orientation: landscape) {
```

This uses iPhone 13 Pro Max viewport as it is one of the biggest mobile viewports on the market: https://blisk.io/devices/details/iphone-13-pro-max